beast/plotting/plot_indiv_fit.py

```diff
@@ -515,8 +515,8 @@ def rectangle_around_axes(bottomleft_ax, topright_ax, pad, ls, label=None):
     filters = [
         "HST_WFC3_F275W",
         "HST_WFC3_F336W",
-        "HST_ACS_WFC_F475W",
-        "HST_ACS_WFC_F814W",
+        "HST_WFC3_F475W",
+        "HST_WFC3_F814W",
```

Comment:
Humm.... this changes the default filters for the plot_indiv_fit.  I think this means that the `phat_small` example will not plot.  Was this intentional?

Reply:
Nope! I didn't mean for that to be in this at all! Will investigate

Reply:
Fixed! (I think)